crates/shared/src/sources/balancer_v2/swap.rs

```diff
@@ -150,19 +151,35 @@ struct BalancesWithIndices {
     balances: Vec<Bfp>,
 }
 
-impl StablePoolRef<'_> {
+impl<'a> StablePoolRef<'a> {
+    /// Composable stable pools have their own BPT tokens as a registered token
+    /// and, when doing "regular swaps" skips the BPT token.
```

Comment:
This wording seems a bit strange. Note that this is duplicated in case you end up changing it.

Reply:
4275e2c636691bf40b1a946a233d374275de904d